cypress/integration/exchange.js

```diff
@@ -1,11 +1,12 @@
 describe("Augmint exchange", function() {
+    let tradeHistoryStartLength = document.querySelectorAll("[data-testid=trade-history] tbody tr").length;
```

Comment:
if you save the tradehistory len here then the tests won't be independent. Ie. try running a last one adding `.only` (after it). Or try to swap the last two.
I would suggest to save it before each test where you need it or just place it in beforeEach.

Reply:
I made a huge refactor